Release 0.21.0rc2 (#23)
dbt-labs/dbt-release: releases/0.21/0.21.rc2.txt

commit: eb079dd818ff8e1b96f2f62beb26ff0a2616d33e
branch: 0.21.latest
version: 0.21.0rc2

> An initial release candidate for v0.21.0.

Reviewer: ```suggestion
A second release candidate for v0.21.0.
```
